be/src/olap/rowset/segment_v2/segment.cpp

```diff
@@ -290,6 +290,10 @@ Status Segment::_load_pk_bloom_filter() {
         return _load_pk_bf_once.call([this] {
             RETURN_IF_ERROR(_pk_index_reader->parse_bf(_file_reader, *_pk_index_meta));
             _meta_mem_usage += _pk_index_reader->get_bf_memory_size();
+            SegmentCache::CacheKey cache_key(_rowset_id, _segment_id);
```

Comment:
It's better to decouple cache management and load index/bloomfilter

Reply:
Sometimes, we might call `_load_pk_bloom_filter()` on `Segment` objects that not in cache. Not all Segment object should in SegmentCache